operator/v1/types_klusterlet.go

```diff
@@ -157,6 +157,44 @@ type RegistrationConfiguration struct {
 	// ManagedCluster when creating only, other actors can update it afterwards.
 	// +optional
 	ClusterAnnotations map[string]string `json:"clusterAnnotations,omitempty"`
+
+	// KubeAPIQPS indicates the maximum QPS while talking with apiserver of hub cluster from the spoke cluster.
+	// If it is set empty, use the default value: 50
+	// +optional
+	// +kubebuilder:default:=50
+	// +kubebuilder:validation:Pattern=^(-?\d+)(.\d+)?$
+	KubeAPIQPS string `json:"qps,omitempty"`
```

Comment:
the json field should be also kubeAPIQPS, why it is not int32 with max?

Reply:
changed to kubeAPIQPS. the QPS is float, but float is not suggested to be used in API, so use string here and added a validation.